content/updates/2017-08-08-max-gov-migration-instructions.md

```diff
@@ -0,0 +1,91 @@
++++
+date = "2017-08-08"
+title = "Announcement: cloud.gov accounts will migrate to OMB MAX"
++++
+
+cloud.gov is migrating all users with native cloud.gov accounts (not users with an agency single sign on provider) to use OMB MAX for authentication to the platform. This page is a guide to what will change and how you can prepare for this transition.
+
+## What’s changing?
+
+Right now, there are two ways to log in to cloud.gov: either you use your agency’s credentials or a cloud.gov account. This change only affects users who have a cloud.gov account.
+
+<img src="/img/login-old.png" alt="the current cloud.gov login screen" width="624" height="622" />
+
+In September 2017, we will switch users with a cloud.gov account to use an OMB MAX account to log in to cloud.gov. We’re making this change both to increase the compliance of cloud.gov with federal standards and law and to make managing your account easier for you and your agency.
+
+## What is OMB MAX?
+
+OMB MAX is a governmentwide collaboration platform with an authentication system agencies can use to log in to other shared services. MAX.gov also enables using a PIV/CAC card to log in to web services.
+
+Though we currently use multi-factor authentication for all cloud.gov accounts, switching to MAX accounts enables agencies which require PIV/CAC cards for login to use cloud.gov more easily. MAX also makes it easier for agencies to meet their own compliance requirements for users of cloud.gov by implementing single sign-on with agency services. For most users, this will provide a better user experience by not having a separate login and second factor to keep track of for cloud.gov.
+
+## What does this mean for me?
+
+If you already use your agency’s single sign-on (SSO) to log in to cloud.gov: there is nothing further you need to do. Currently agencies using SSO are EPA, GSA, and FDIC. These customers will continue to log in as usual.
+
+**If you use a cloud.gov account and already have a MAX account**, double check that you can log in successfully to MAX and have a Secure+ SMS 2-factor device paired with your account.
+
+**If you use a cloud.gov account to log in and don’t have a MAX account,** you need to create one. Once we switch, you’ll be required to use that account to login to cloud.gov and all associated services.
+
+Create a MAX account by going to login.max.gov and clicking the green Register Now button in the top right corner or your agency’s logo. Make sure to add a 2-factor device to your account. The rest of this page has more detailed instructions for creating accounts if you need them.
+
+### Log in to cloud.gov with MAX
```

Comment:
I'd suggest removing this section for now since readers won't be able to do this yet - we can publish a followup with this info when it's possible.